scavenge/x/scavenge/internal/types/params.go

```diff
@@ -0,0 +1,56 @@
+package types
```

Comment:
can be removed

Reply:
think i need some of these functions even tho they dumb dumb